manifests/init.pp

```diff
@@ -211,6 +214,7 @@
   Pulpcore::ChecksumTypes $allowed_content_checksums = ['sha224', 'sha256', 'sha384', 'sha512'],
   String[1] $remote_user_environ_name = 'HTTP_REMOTE_USER',
   Integer[0] $worker_count = min(8, $facts['processors']['count']),
+  Optional[Integer[0]] $worker_ttl = undef,
```

Comment:
```suggestion
  Optional[Integer[0]] $worker_ttl = 30,
```

The default value for `WORKER_TTL` in Pulp looks like 30 seconds, so this recommendation should be functionally equivalent but improves discoverability for users by making that explicit.

Reply:
```suggestion
  Integer[0] $worker_ttl = 30,
```

Keeping with the theme, let's make it not optional if there will be a default.

Reply:
My intent here was let's not hard-code Pulp's defaults. If Pulp changes the default, then we have to change this. If we set this to undef, then we let Pulp control the default.